pyproject.toml

```diff
@@ -1,6 +1,77 @@
+[project]
+name = "pudl_archiver"
+version = "0.2.0"
+authors = [{name = "PUDL", email = "pudl@catalyst.coop"}]
+requires-python = ">=3.10,<3.11"
+
+dependencies = [
+    "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git@dev",
+    "coloredlogs~=15.0",
+    "factory_boy>=2.12",
+    "feedparser>=6.0",
+    "tqdm>=4.64",
+    "catalystcoop.arelle-mirror==1.3.0",
```

Comment:
No longer required. If we need arelle we can depend on the production release of arelle (see main PUDL repo dependencies)

Reply:
There's some `import arelle` [here](https://github.com/catalyst-cooperative/pudl-archiver/blob/daz/pyproject-toml/src/pudl_archiver/archivers/ferc/xbrl.py#L18) though I bet it's worth giving production arelle a shot.